bench_runner/templates/_benchmark.src.yml

```diff
@@ -48,9 +48,6 @@ name: _benchmark
       benchmarks:
         description: "Benchmarks to run (comma-separated; empty runs all benchmarks)"
         type: string
-      pgo:
-        description: "Build with PGO"
-        type: boolean
```

Comment:
Out of curiosity, what effect does removing this do?

Reply:
Github has a limit of only 10 options on a `workflow-dispatch` trigger (that's the thing that makes a drop down of options in the web interface).  In order to accommodate one more checkbox for clang, I had to remove this one.  We never should be benchmarking without PGO anyway, so I thought this was the safest to remove.

![image](https://github.com/user-attachments/assets/cc88b5bc-8f5d-4927-983b-15ac8f2fc234)

Reply:
Ah ok that makes sense. Thanks! I think the next thing to remove if we run out of options again would probably be the tier 2 interpreter since at this point we probably don't care about its performance.